styles/globals.css

```diff
@@ -177,3 +177,36 @@ a {
 .transparency-solution {
   width: 48%;
 }
+
+.navbar-dark .navbar-toggler {
+  border: none;
+}
+
+.navbar-dark .navbar-toggler-icon {
+  background-image: url("data:image/svg+xml,%3csvg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 30 30'%3e%3cpath stroke='rgba(255,255,255, 1)' stroke-linecap='round' stroke-miterlimit='10' stroke-width='2' d='M4 7h22M4 15h22M4 23h22'/%3e%3c/svg%3e");
+}
```

Comment:
are these used?

Reply:
Yes those are custom styles set for the navbar hamburger specifically.  The first gets rid of the bootstrap default border and the second changes the stripes on the hamburger icon from the bootstrap default grey to white.  This is the hamburger looks like without those styles: 
![screencapture-localhost-3000-2022-05-17-21_57_39](https://user-images.githubusercontent.com/83320256/168942050-7f6fb900-6984-4cdb-b81a-40b8bdaac7b6.jpeg)